extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/include.config.html

```diff
@@ -11,56 +11,54 @@
 								<div ng-switch-when="input">
 									<div ng-if="(parameter.type=='INTEGER' || parameter.type=='DECIMAL')">
 										<label class="config-textInput" for="{{parameter.name}}">{{parameter.label}}</label>
-										<md-input-container ng-class="{'md-input-invalid':form.configForm[parameter.name].$invalid}" class="col-xs-12 configContainer">
-										    <input ng-init="focus=false" ng-pattern="parameter.pattern" name="{{parameter.name}}" ng-readonly="parameter.readOnly" type="number" min="{{parameter.min}}" max="{{parameter.max}}" step="{{parameter.stepsize}}" ng-model="configuration[parameter.name]" ng-required="parameter.required" type="{{parameter.inputType}}" ng-focus="focus=true" ng-blur="focus=false">
+										<md-input-container ng-class="{'md-input-invalid':form.configForm[parameter.name].$invalid}" class="col-xs-12 configContainer no-animation"> <input ng-init="focus=false" ng-pattern="parameter.pattern" name="{{parameter.name}}" ng-readonly="parameter.readOnly" type="number" min="{{parameter.min}}" max="{{parameter.max}}" step="{{parameter.stepsize}}" ng-model="configuration[parameter.name]" ng-required="parameter.required" type="{{parameter.inputType}}" ng-focus="focus=true" ng-blur="focus=false">
```

Comment:
Yes, it would stay like that until you format it again. In that case you will have to manually change it every time you use the default ESH format settings. A better solution hence would be to change the settings so that the formatter itself puts the line breaks where necessary.